pkg/kubelet/volume_host.go

```diff
@@ -74,7 +75,8 @@ func NewInitializedVolumePluginMgr(
 			csiDriverInformer := informerFactory.Storage().V1beta1().CSIDrivers()
 			csiDriverLister = csiDriverInformer.Lister()
 			csiDriversSynced = csiDriverInformer.Informer().HasSynced
-
+			csiNodeInformer := informerFactory.Storage().V1beta1().CSINodes()
```

Comment:
This should be protected by CSINodeInfo feature gate

Reply:
This also should export a HasSynced

Reply:
It seems HasSynced is not used for the moment.
Can it be added when it is used ?

thanks

Reply:
We should wait on HasSynced during initialization so that we are not accessing the informer in an inconsistent state.